README.md

```diff
@@ -37,10 +37,10 @@ type Car struct {
 
 owners := []string{}
 
-From(cars).Where(func(c interface{}) bool {
-	return c.(Car).year >= 2015
-}).Select(func(c interface{}) interface{} {
-	return c.(Car).owner
+From(cars).Where(func(c Car) bool {
+	return c.year >= 2015
+}).Select(func(c Car) interface{} {
```

Comment:
how about `}).Select(func(c Car) string {` ? It seems that the only reason you have left `interface {}` here, is for validation purposes. So, what do you think about getting rid of validation for output params in order to make syntax better?